### PR TITLE
Doc wollok game #863

### PR DESCRIPTION
**Pull request by PalumboN**
Please check my english. And comment if something some method need more explication. Thanks!

I rename the accessors following the convention. I'll change the examples soon.

### CODE REVIEW

File: org.uqbar.project.wollok.lib/src/wollok/lib.wlk [hunk 1]
```diff
@@ -180,7 +180,7 @@ object game {
 	 * Adds an object to the board for drawing it.
 	 * That object should known a position.
 	 */
-	method addVisual(element) native
+	method addVisual(visual) native
```
Comment (Juancete) [inline org.uqbar.project.wollok.lib/src/wollok/lib.wlk]: Perhaps can be 

method addVisual(domainObject) native  ??
method addVisual(visualObject) native  ??
method addVisual(objectToView) native  ??

I'm not sure ....

File: org.uqbar.project.wollok.lib/src/wollok/lib.wlk [hunk 2]
```diff
@@ -228,6 +228,7 @@ object game {
 	 * Removes all objects on board and configurations (colliders, keys, etc).
 	 */	
 	method clear() native
+	method colliders(visual) native
```
Comment (Juancete) [inline org.uqbar.project.wollok.lib/src/wollok/lib.wlk]: Same as other comments.... visual? domainObject? objectToView?

**Comment by javierfernandes** [comment]
addVisual(positionable) ?

If it only needs a position :P

**Comment by npasserini** [comment]
+1

**Comment by Juancete** [comment]
Likeeeeeeeee

**Comment by PalumboN** [comment]
Done! There exist two options for work with the position:

1. The domain object knows its position (by attribute or getter). In this case, we name it _positionable_.

2. The domain object doesn't know its position, WG receives both object (domain and positionn) and it makes the relation. In this case is not a _positionable_, it's an simple object like other. In this case we are calling _element_ when it's trying to be rendered and _visual_ when it already should be rendering.

**Comment by PalumboN** [comment]
Ah! And I remove some old logs. A related issue is here: https://github.com/uqbar-project/wollok/issues/955

File: org.uqbar.project.wollok.lib/src/wollok/lib.wlk [hunk 3]
```diff
@@ -202,7 +202,7 @@ object game {
 	/**
 	 * Removes an object from the board for stop drawing it.
 	 */
-	method removeVisual(element) native
+	method removeVisual(visual) native
```
Comment (Juancete) [inline org.uqbar.project.wollok.lib/src/wollok/lib.wlk]: Must be positionable... right?

Comment (PalumboN) [inline org.uqbar.project.wollok.lib/src/wollok/lib.wlk]: Not necessary, for the point 2 of my comment :P

Comment (Juancete) [inline org.uqbar.project.wollok.lib/src/wollok/lib.wlk]: haaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

:cocktail:

**Pull request merged**